packages/back-end/test/middleware/authenticateApiRequestMiddleware.test.ts

```diff
@@ -201,8 +207,49 @@ describe("REST API auth middleware", () => {
           environments,
           project,
           teams: [],
+          superAdmin: false,
         });
       }).toThrowError();
     });
+
+    it("should throw an error when user is not part of an org", () => {
+      const permission: Permission = "readData";
+      const project = undefined;
+      const environments = ["production", "staging"];
+
+      expect(() => {
+        verifyApiKeyPermission({
+          apiKey: {
+            ...userKey,
+            userId: "u_unknown_id",
+          },
+          permission,
+          organization,
+          environments,
+          project,
+          teams: [],
+          superAdmin: false,
+        });
+      }).toThrowError();
+    });
+
+    it("should allow superAdmins to readData even when not part of an org", () => {
+      const permission: Permission = "readData";
+      const project = undefined;
+      const environments = ["production", "staging"];
+
+      verifyApiKeyPermission({
+        apiKey: {
+          ...userKey,
+          userId: "u_unknown_id",
+        },
+        permission,
+        organization,
+        environments,
+        project,
+        teams: [],
+        superAdmin: true,
+      });
```

Comment:
add'l test: it should allow superAdmins full permissions/member permissions if they are a part of org